bin/build.sh

```diff
@@ -451,4 +454,17 @@ then
   puts "Regression tests complete"
 fi
 
+if [ ${INSTALL_OBJECTIVEC} -eq 1 ]
+then
+  puts "Installing GNU Objective-C for linux"
+
+  sudo apt-get -y install gobjc gnustep gnustep-make gnustep-common gnustep-devel
+  # The version numbers here will have to change by OS.
+  sudo ln -s /usr/lib/gcc/x86_64-linux-gnu/4.8/include/objc /usr/local/include/objc
```

Comment:
May I ask why a symbolic link is required here?

Reply:
The way that apt installs the objc libraries, it puts the objc.h file into a place where clang can't find it. Rather than make clang include this directory, I symlink it into the include folder so it sees it by default (this was the internet's solution to this error: https://ubuntuforums.org/showthread.php?t=1412542)

Reply:
I think we need to fix this otherwise SMACK's Objective-C support cannot be locally installed. Maybe this link is helpful, https://stackoverflow.com/questions/11475124/unable-to-find-objc-h-during-clang-compile.